### ISSUE TITLE
Update default Kabanero pipelines to leverage the same Pipelines using new Kabanero Symbolic for CollectionName

### ISSUE THREAD

**Issue by stephenkinder** [kabanero-io/kabanero-pipelines#40]
Exploit the new capability introduced via https://github.com/kabanero-io/kabanero-operator/issues/114 and rewire all of the pipelines delivered for the Kabanero collection to point at the same build pipeline.

**Comment by aadeshpa** [comment]
For testing this issue changes, there are 2 other things that needed to be present 
1. https://github.com/kabanero-io/kabanero-operator/issues/114 : Kyle's changes in kabanero-operator
2. @steven.groeger : changes for pulling the common generalized pipeline and tasks from the collections repository location 'collections/incubator/common/pipelines/default/' so kabanero operator will activate the files.

In order to test all these things together we needed build of collections and dummy release link from @steven.groeger so we could test it in the environment we have.  Once we get that latest dummy release with our updated pipeline and task files we can proceed further

**Comment by aadeshpa** [comment]
After we got the updated build of the changes from @steven.groeger we verified that by running in our environment and on applying the yaml (https://raw.githubusercontent.com/kabanero-io/kabanero-operator/master/config/samples/full.yaml) after substituting the release url  value in it as (https://github.com/kabanero-io/collections/releases/download/v0.1.1.beta1/kabanero-index.yaml)[given by Steven] , the collections were re-generated however the text substitution has some bug. Attaching the screenshots of the collections describe commands.

**Comment by aadeshpa** [comment]
<img width="1387" alt="Screen Shot 2019-09-05 at 5 13 56 PM" src="https://user-images.githubusercontent.com/53223993/64381666-bf079280-d001-11e9-932b-e83a6f72a5ce.png">
<img width="1396" alt="Screen Shot 2019-09-05 at 5 12 44 PM" src="https://user-images.githubusercontent.com/53223993/64381667-bf079280-d001-11e9-9e35-96e4e8ba2959.png">
<img width="1388" alt="Screen Shot 2019-09-05 at 5 11 53 PM" src="https://user-images.githubusercontent.com/53223993/64381668-bf079280-d001-11e9-9ce7-2411ad54d37b.png">
<img width="1383" alt="Screen Shot 2019-09-05 at 5 04 06 PM" src="https://user-images.githubusercontent.com/53223993/64381669-bf079280-d001-11e9-9372-8d5ff18a9750.png">

**Comment by aadeshpa** [comment]
We are on standby for @kschloss: to provide us the kabanero-operator fix for correct collection-names substitution, so we could go ahead and test in our test environment, whether all collections pipelines and tasks are activated correctly. cc: @marikaj

**Comment by tseelbach** [comment]
Adding to what Ashish commented,  we are blocked  https://github.com/kabanero-io/kabanero-operator/issues/114

**Comment by aadeshpa** [comment]
We verified after @kschloss fix 
1. We build the collections manually with our task and pipeline file changes and uploaded the artifacts to the dummy test release
2.Then we applied that new release in our environment and then all the collections and pipelines got activated correctly with correct substitution.
3. Finally I ran nodejs express pipeline and it worked correctly proving that @kschloss  fix is correct and our changes are correct. 

Created below two pull request and merged it
1. https://github.com/kabanero-io/collections/pull/62
2. https://github.com/kabanero-io/collections/pull/68

Marking it as done.
